raw-cms-app/src/modules/core/views/lambda-details-view/lambda-details-view.js

```diff
@@ -31,6 +32,8 @@ const _LambdaDetailsView = async (res, rej) => {
         this.title = isNew
```

Comment:
No need for `title` property on component, see comment on "raw-cms-app/src/modules/core/views/collection-table-view/collection-table-view.js".

Reply:
removed title var from data lifecycle,